spec/features/user_edits_information_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+feature "user edits account settings", %(
+  As a user
+  I want to edit my account settings
+  So that I can change my information
+
+  Acceptance Criteria
+  [x] I must be able to change my password
+  [x] I must be able to change my username
+  [x] I must be able to change my email
+  [x] I must be able to change my avatar
+  [x] I must see the link to account settings when I am signed in
+) do
+
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+  end
+
+  scenario 'user changes password and email' do
+    click_link 'Account Settings'
+    expect(current_path).to eq edit_user_registration_path
+
+    new_password = "boogeyman"
+    new_email = "boogeyman@gmail.com"
```

Comment:
Nice use of setting the strings as variables in the tests so that your assertions use fixed strings.